Makefile

```diff
@@ -32,8 +32,9 @@ VENDOR_ENV := GO15VENDOREXPERIMENT=1
 SERVICES := \
 	m3dbnode
 
-TOOLS := \
-	read_ids
+TOOLS :=   \
+	read_ids \
+	index_reader
```

Comment:
For slightly more consistent naming maybe it makes sense to make this `read_index_ids`?